Libplanet/Action/ActionEvaluation.cs

```diff
@@ -53,5 +57,7 @@ public class ActionEvaluation
         /// An exception that had risen during evaluation.
         /// </summary>
         public Exception? Exception { get; }
+
+        public List<string> Logs { get; }
```

Comment:
Could you write an XML comment here?

Reply:
I wrote an XML comment in [d853af3](https://github.com/planetarium/libplanet/pull/2505/commits/d853af33fd905bd2788281cda6356a3062d7182e) commit.